<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Toynote</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="notesView">
      <property name="minimumSize">
       <size>
        <width>382</width>
        <height>0</height>
       </size>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Notebook"/>
    <addaction name="actionOpen_Notebook"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose_Notebook"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionVisit_Website"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Notebook"/>
   <addaction name="actionOpen_Notebook"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Note"/>
   <addaction name="actionDelete_Notes"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Notebook">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Notebook-new.svg</normaloff>:/icons/Notebook-new.svg</iconset>
   </property>
   <property name="text">
    <string>New Notebook</string>
   </property>
   <property name="toolTip">
    <string>Create a new notebook</string>
   </property>
  </action>
  <action name="actionNew_Note">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Note-add.svg</normaloff>:/icons/Note-add.svg</iconset>
   </property>
   <property name="text">
    <string>New Note</string>
   </property>
   <property name="toolTip">
    <string>Add a new note</string>
   </property>
  </action>
  <action name="actionDelete_Notes">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Note-remove.svg</normaloff>:/icons/Note-remove.svg</iconset>
   </property>
   <property name="text">
    <string>Delete Notes</string>
   </property>
   <property name="toolTip">
    <string>Delete the selected note(s)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Toynote</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Notebook-save.svg</normaloff>:/icons/Notebook-save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save the notebook</string>
   </property>
  </action>
  <action name="actionOpen_Notebook">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Notebook-open.svg</normaloff>:/icons/Notebook-open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Notebook</string>
   </property>
   <property name="toolTip">
    <string>Open a notebook</string>
   </property>
  </action>
  <action name="actionClose_Notebook">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close the notebook</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Notebook-save-as.svg</normaloff>:/icons/Notebook-save-as.svg</iconset>
   </property>
   <property name="text">
    <string>Sa&amp;ve As</string>
   </property>
   <property name="toolTip">
    <string>Save the notebook to a chosen file</string>
   </property>
  </action>
  <action name="actionVisit_Website">
   <property name="text">
    <string>Visit Website</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>displayAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveNotebook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Notebook</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openNotebook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose_Notebook</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeNotebook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_Notebook</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newNotebook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_As</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveNotebookAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_Note</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newNote()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete_Notes</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteNotes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notesView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>editNote(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVisit_Website</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>visitWebsite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displayAbout()</slot>
  <slot>saveNotebook()</slot>
  <slot>openNotebook()</slot>
  <slot>newNotebook()</slot>
  <slot>closeNotebook()</slot>
  <slot>saveNotebookAs()</slot>
  <slot>newNote()</slot>
  <slot>deleteNotes()</slot>
  <slot>visitWebsite()</slot>
  <slot>exit()</slot>
  <slot>editNote(QModelIndex)</slot>
 </slots>
</ui>
